app/javascript/app.spec.js

```diff
@@ -0,0 +1,14 @@
+import { shallowMount } from '@vue/test-utils';
+import App from 'app';
+
+describe('App', () => {
+  test('is a Vue instance', () => {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'test' is not defined      no-undef